File: daily-coding/coding-bat/java/makes10.java
// Given 2 ints, a and b, return true if one if them is 10 or if their sum is 10.

public boolean makes10(int a, int b) {
  boolean ten = false;
  if (a == 10 || b == 10 || ((a + b)) == 10){
    ten = true;
  }
  return ten;
}

File: daily-coding/coding-bat/python/makes10.py
# Given 2 ints, a and b, return True if one if them is 10 or if their sum is 10.

def makes10(a, b):
  summation = a + b
  is10 = False
  if summation == 10 or a == 10 or b == 10:
    is10 = True
  return is10
